libethash-cuda/CUDAMiner.cpp

```diff
@@ -17,6 +17,8 @@ along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 /** @file CUDAMiner.cpp
 * @author Gav Wood <i@gavwood.com>
 * @date 2014
+* @author MariusVanDerWijden
```

Comment:
It's better to remove entire comment. Git should be enough to find contributors.

Reply:
OK